add a part in the readme about metadata (#10)
MarkPflug/Sylvan.BuildTools.Resources: README.md

}
```

> You can generate controll the resource generation specifying a custom namespace, or the visibility of the generated class:

Reviewer: ```suggestion
You can control the resource generation specifying a custom namespace, and the visibility of the generated class:
```
